src/decision/src/GpsManager.cpp

```diff
@@ -1,29 +1,11 @@
 /*
  * Created By: Gareth Ellis
- * Created On: October 20, 2016
+ * Created On: May 17, 2017
  * Description: Manages the GPS waypoints we are given
  *              (publishing the next one once we've arrived
  *              at the current one)
  */
```

Comment:
Same thing here, basically new file, probably want to just review as a new submission, rather then via the diff.